### PR TITLE
Feature/no maap cfg

### PR DESCRIPTION
**Pull request by grallewellyn**
This PR merged in Sujen's changes to remove the need to track maap.cfg file

We will need to deploy the changes to the devfiles when we merge this PR so that the `MAAP_API_HOST` variable is present. `MAAP_API_HOST` points to the ops api by default so it will need to be added to the devfiles for dit and uat

### CODE REVIEW

File: devfiles/isce3/devfile/devfile.yaml [hunk 1]
```diff
@@ -36,6 +36,8 @@ components:
                 env:
                 - name: DOCKERIMAGE_PATH
                   value: 'mas.dit.maap-project.org/root/maap-workspaces/base_images/isce3:develop'
+                - name: MAAP_API_HOST
+                  value: https://ade.dit.maap-project.org/api
```
Comment (sujen1412) [inline devfiles/isce3/devfile/devfile.yaml:40]: This value should be `ade.dit.maap-project.org`

File: devfiles/pangeo/devfile/devfile.yaml [hunk 2]
```diff
@@ -36,6 +36,8 @@ components:
                 env:
                 - name: DOCKERIMAGE_PATH
                   value: 'mas.dit.maap-project.org/root/maap-workspaces/base_images/pangeo:develop'
+                - name: MAAP_API_HOST
+                  value: https://ade.dit.maap-project.org/api
```
Comment (sujen1412) [inline devfiles/pangeo/devfile/devfile.yaml:40]: This value should be `ade.dit.maap-project.org`

File: devfiles/python/devfile/devfile.yaml [hunk 3]
```diff
@@ -36,6 +36,8 @@ components:
                 env:
                 - name: DOCKERIMAGE_PATH
                   value: 'mas.dit.maap-project.org/root/maap-workspaces/base_images/vanilla:develop'
+                - name: MAAP_API_HOST
+                  value: https://ade.dit.maap-project.org/api
```
Comment (sujen1412) [inline devfiles/python/devfile/devfile.yaml:40]: This value should be `ade.dit.maap-project.org`

File: devfiles/r/devfile/devfile.yaml [hunk 4]
```diff
@@ -36,6 +36,8 @@ components:
                 env:
                 - name: DOCKERIMAGE_PATH
                   value: 'mas.dit.maap-project.org/root/maap-workspaces/base_images/r:develop'
+                - name: MAAP_API_HOST
+                  value: https://ade.dit.maap-project.org/api
```
Comment (sujen1412) [inline devfiles/r/devfile/devfile.yaml:40]: This value should be `ade.dit.maap-project.org`

**Review by sujen1412 — CHANGES_REQUESTED** [review]

**Review by sujen1412 — DISMISSED** [review]

**Review by sujen1412 — APPROVED** [review]

**Comment by sujen1412** [comment]
I just realized that I did not read the env value correctly. The API host is supposed to point to api.*.maap-project.org and not ade.*.maap-project.org. Sorry missed this.

**Comment by grallewellyn** [comment]
@sujen1412 Created a PR to address this here: https://github.com/MAAP-Project/maap-workspaces/pull/126

**Pull request merged**